modules/core/src/v2/environments.ts

```diff
@@ -103,165 +103,115 @@ const devBase: EnvironmentTemplate = Object.assign({}, testnetBase, {
-    },
-    mainnetBase
-  ),
-  staging: Object.assign(
-    {
-      uri: 'https://staging.bitgo.com',
-      stellarFederationServerUrl: 'https://staging.bitgo.com/api/v2/xlm/federation',
-    },
-    mainnetBase
-  ),
-  rmgStaging: Object.assign(
-    {
-      uri: 'https://rmgstaging.bitgo.com',
-      stellarFederationServerUrl: 'https://rmgstaging.bitgo.com/api/v2/xlm/federation',
-    },
-    mainnetBase
-  ),
-  test: Object.assign(
-    {
-      uri: 'https://test.bitgo.com',
-      stellarFederationServerUrl: 'https://test.bitgo.com/api/v2/txlm/federation',
-    },
-    testnetBase
-  ),
-  rmgTest: Object.assign(
-    {
-      uri: 'https://rmgtest.bitgo.com',
-      stellarFederationServerUrl: 'https://rmgtest.bitgo.com/api/v2/txlm/federation',
-    },
-    testnetBase
-  ),
-  dev: Object.assign(
-    {
-      uri: 'https://webdev.bitgo.com',
-      stellarFederationServerUrl: 'https://webdev.bitgo.com/api/v2/txlm/federation',
-    },
-    devBase
-  ),
-  latest: Object.assign(
-    {
-      uri: 'https://latest.bitgo.com',
-      stellarFederationServerUrl: 'https://latest.bitgo.com/api/v2/txlm/federation',
-    },
-    devBase
-  ),
-  rmgLatest: Object.assign(
-    {
-      uri: 'https://rmglatest.bitgo.com',
-      stellarFederationServerUrl: 'https://rmglatest.bitgo.com/api/v2/txlm/federation',
-    },
-    devBase
-  ),
-  rmgDev: Object.assign(
-    {
-      uri: 'https://rmgwebdev.bitgo.com',
-      stellarFederationServerUrl: 'https://rmgwebdev.bitgo.com/api/v2/txlm/federation',
-    },
-    devBase
-  ),
-  local: Object.assign(
-    {
-      uri: 'https://localhost:3000',
-      stellarFederationServerUrl: 'https://localhost:3000/api/v2/txlm/federation',
-    },
-    devBase
-  ),
-  localNonSecure: Object.assign(
-    {
-      uri: 'http://localhost:3000',
-      stellarFederationServerUrl: 'http://localhost:3000/api/v2/txlm/federation',
-    },
-    devBase
-  ),
-  mock: Object.assign(
-    {
-      uri: 'https://bitgo.fakeurl',
-      smartBitApiBaseUrl: 'https://testnet-api.smartbit.fakeurl/v1',
-      bchExplorerBaseUrl: 'https://test-bch-insight.bitpay.fakeurl/api',
-      stellarFederationServerUrl: 'https://bitgo.fakeurl/api/v2/txlm/federation',
-      etherscanBaseUrl: 'https://kovan.etherscan.fakeurl',
-      ltcExplorerBaseUrl: 'http://explorer.litecointools.fakeurl/api',
-      zecExplorerBaseUrl: 'https://explorer.testnet.z.fakeurl/api',
-      dashExplorerBaseUrl: 'https://testnet-insight.dashevo.fakeurl/insight-api',
-    },
-    devBase
-  ),
-  rmgLocal: Object.assign(
-    {
-      uri: 'https://rmglocalhost:3000',
-      stellarFederationServerUrl: 'https://rmglocalhost:3000/api/v2/txlm/federation',
-    },
-    devBase
-  ),
-  rmglocalNonSecure: Object.assign(
-    {
-      uri: 'http://rmglocalhost:3000',
-      stellarFederationServerUrl: 'http://rmglocalhost:3000/api/v2/txlm/federation',
-    },
-    devBase
-  ),
-  custom: Object.assign(
```

Comment:
just curious, what is "custom" used for?

Reply:
Custom is used in ms branch deploys, and some other circumstances where the endpoint isn't necessarily known until runtime (eg, dynamic proxy, running SDK against bitgo-express, etc).